lighthouse-plugin-ad-speed-insights/package.json

```diff
@@ -16,7 +16,7 @@
   ],
   "author": "",
   "dependencies": {
-    "lighthouse": "^4.1.0",
+    "lighthouse": "^4.2.0",
```

Comment:
not a change to be made now, but we might want to do this as a `peerDependency` eventually? But it really depends on how it's intended to be installed, I guess

Reply:
Agreed, moved to https://github.com/googleads/ad-speed-insights/issues/8